core/src/commonMain/kotlin/com/xebia/functional/xef/llm/assistants/RunDeltaEvent.kt

```diff
@@ -0,0 +1,27 @@
+package com.xebia.functional.xef.llm.assistants
+
+enum class RunDeltaEvent {
```

Comment:
Should this be private inside of `RunDelta.kt`? Or is there a need to export this?